<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirelocationPredict</class>
 <widget class="QWidget" name="FirelocationPredict">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1608</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FSRSP</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="imageinput">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>image input</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_left">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="leftimage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 199, 199)</string>
              </property>
              <property name="text">
               <string>Please select a image</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_left_button">
            <item>
             <widget class="QPushButton" name="leftselect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Select the left wall image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="leftcancel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>clean</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_central">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="centralimage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 199, 199)</string>
              </property>
              <property name="text">
               <string>Please select a image</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_central_button">
            <item>
             <widget class="QPushButton" name="centralselect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Select the central wall image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="centralcancel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>clean</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_right">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="rightimage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>300</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(199, 199, 199)</string>
              </property>
              <property name="text">
               <string>Please select a image</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="rightselect">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Select the right wall image</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="rightcancel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>clean</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="verticalLayout_operatelog">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>10000</height>
        </size>
       </property>
       <property name="title">
        <string>Operation Record</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextBrowser" name="operatelog"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="cleanlog_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>clean</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_output">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QGroupBox" name="sootpatternextract">
       <property name="title">
        <string>Soot Pattern Extraction</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_extractimage">
          <item>
           <widget class="PlotWidget" name="leftextractimage">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlotWidget" name="centralextractimage">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PlotWidget" name="rightextractimage">
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_extractbutton">
          <item>
           <widget class="QPushButton" name="extract_btn">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Extracted features</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="firelocationpredict">
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>10000</height>
        </size>
       </property>
       <property name="title">
        <string>Fire source location prediction</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_firelocationimage">
          <item>
           <widget class="PlotWidget" name="firelocationimage">
            <property name="maximumSize">
             <size>
              <width>500</width>
              <height>500</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_labellocation">
          <item>
           <widget class="QLabel" name="label_location">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>Coordinates: </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_predict">
          <item>
           <widget class="QPushButton" name="predict_btn">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="text">
             <string>Predict</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
